src/Symfony/Component/Cache/Adapter/AbstractTagAwareAdapter.php

```diff
@@ -35,7 +35,7 @@ abstract class AbstractTagAwareAdapter implements TagAwareAdapterInterface, TagA
     use AbstractAdapterTrait;
     use ContractsTrait;
 
-    private const TAGS_PREFIX = "\1tags\1";
+    protected const TAGS_PREFIX = "\0tags\0";
```

Comment:
Why is this changing the value of the constant ? IIRC, we stopped using `\0` because NUL bytes were causing issues with some storages.

Reply:
agreed was wrongly merged from 5.4